packages/cli/src/controllers/telemetry.controller.ts

```diff
@@ -0,0 +1,59 @@
+import { GlobalConfig } from '@n8n/config';
+import { AuthenticatedRequest } from '@n8n/db';
+import { Get, Post, RestController } from '@n8n/decorators';
+import { NextFunction, Response } from 'express';
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware';
+
+@RestController('/telemetry')
+export class TelemetryController {
+	proxy;
+
+	constructor(private readonly globalConfig: GlobalConfig) {
+		this.proxy = createProxyMiddleware({
+			target: this.globalConfig.diagnostics.frontendConfig.split(';')[1],
+			changeOrigin: true,
+			pathRewrite: {
+				'^/proxy/': '/', // /proxy/v1/track -> /v1/track
+			},
+			on: {
+				proxyReq: (proxyReq, req) => {
+					proxyReq.removeHeader('cookie');
+					fixRequestBody(proxyReq, req);
+					return;
+				},
+			},
+		});
+	}
+
+	@Post('/proxy/:version/track', { skipAuth: true })
+	async track(req: AuthenticatedRequest, res: Response, next: NextFunction) {
+		await this.proxy(req, res, next);
+	}
+
+	@Post('/proxy/:version/identify', { skipAuth: true })
+	async identify(req: AuthenticatedRequest, res: Response, next: NextFunction) {
+		await this.proxy(req, res, next);
+	}
+
+	@Post('/proxy/:version/page', { skipAuth: true })
+	async page(req: AuthenticatedRequest, res: Response, next: NextFunction) {
+		await this.proxy(req, res, next);
+	}
+	@Get('/rudderstack/sourceConfig', { skipAuth: true })
+	async sourceConfig() {
+		const response = await fetch('https://api-rs.n8n.io/sourceConfig', {
```

Comment:
Instead of hardcoding this (it was in the frontend), add it to `N8N_DIAGNOSTICS_CONFIG_FRONTEND` env variable.

Reply:
Under which circumstances would one want to change this variable? 🤔

Reply:
This should not ever change unless we move the source config to another domain which I doubt. It was more bout moving it to the related configuration env variable instead of having these here and the other two values in the env variable.